specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2023-09-01-preview/operationStatuses.json

```diff
@@ -54,6 +54,54 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.ApiManagement/locations/{location}/operationResults/{operationId}": {
+      "get": {
+        "tags": [
+          "OperationResults"
+        ],
+        "description": "Returns operation results for long running operations executing DELETE or PATCH on the resource.",
+        "operationId": "OperationsResults_Get",
+        "x-ms-examples": {
+          "DeleteOperationResult": {
+            "$ref": "./examples/ApiManagementGetOperationResult.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/LocationParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/OperationIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully retrieved the operation result."
+          },
+          "202": {
```

Comment:
>202 [](http://example.com/codeflow?start=11&length=3)

202 is used generally when a long running request is accepted.

Reply:
Yes, we will be returning this URL in the response to PATCH/DELETE
https://github.com/cloud-and-ai-microsoft/resource-provider-contract/blob/503f887478f2b7383343da848edcfbc2e029780e/v1.0/async-api-reference.md#202-accepted-and-location-headers

and it is not expected to have response body.